10-ambarom-vignette.Rmd

```diff
@@ -47,7 +49,7 @@ library(ggpattern)
 library(osfr)
 source("helper-fun/helper-functions.R")
 
-ambarom_in <- read_rds_tsr("lapop_2021.rds")
+ambarom_in <- read_rds_tsr("lapop_2021.rds") 
```

Comment:
we mention above that the data is not available and they have to download it themselves.  should we mask this somehow then?